calico/getting-started/kubernetes/vpp/openshift.mdx

```diff
@@ -0,0 +1,211 @@
+```
+
+
+### Optionally provide additional configuration
+
+You may want to provide Calico or the VPP dataplane with additional configuration at install time. For example, BGP configuration or peers.
+You can use a Kubernetes ConfigMap with your desired Calico resources in order to set configuration as part of the installation.
+If you do not need to provide additional configuration, you can skip this section.
+
+To include [Calico resources](../../../reference/resources/index.mdx) during installation, edit `manifests/02-configmap-calico-resources.yaml` in order to add your own configuration.
+
+In order to customize the configuration of the VPP dataplane, edit `manifests/03-configmap-calico-vpp-resources.yaml`. For details refer to the **Configuration options** section of [Getting Started](./getting-started.mdx).
+
+:::note
+
+If you have a directory with the Calico resources, you can create the file with the command:
+
+```
+oc create configmap -n tigera-operator calico-resources \
+--from-file=<resource-directory> --dry-run -o yaml \
+manifests/02-configmap-calico-resources.yaml
+```
+
+With recent versions of `oc` it is necessary to have a kubeconfig configured or add `--server='127.0.0.1:443'`
+even though it is not used.
+
+:::
+
+:::note
+
+If you have provided a `calico-resources` ConfigMap and the tigera-operator pod fails to come up with `Init:CrashLoopBackOff`,
+check the output of the init-container with `oc logs -n tigera-operator -l k8s-app=tigera-operator -c create-initial-resources`.
+
+:::
+
+### Create the cluster
+
+Start the cluster creation with the following command and wait for it to complete.
+
+```bash
+openshift-install create cluster
+```
+
+Once the above command is complete, you can verify {{prodname}} VPP is installed by verifying the components are available with the following command.
+
+```
+oc get tigerastatus
+```
+
+:::note
+
+To get more information, add `-o yaml` to the above command.
+
+:::
+
+### Optionally integrate with Operator Lifecycle Manager (OLM)
+
+In OpenShift Container Platform, the [Operator Lifecycle Manager](https://docs.openshift.com/container-platform/4.4/operators/understanding_olm/olm-understanding-olm.html#olm-overview_olm-understanding-olm) helps
+cluster administrators manage the lifecycle of operators in their cluster. Managing the {{prodname}}
+operator with OLM gives administrators a single place to manage operators.
+
+1. To register the running {{prodname}} operator with OLM, first you will need to create an OperatorGroup for the operator:
+
+```bash
+oc apply -f - <<EOF
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: tigera-operator
+  namespace: tigera-operator
+spec:
+  targetNamespaces:
+    - tigera-operator
+EOF
+```
+
+2. Create a Subscription to the operator. By subscribing to the operator package, the {{prodname}} operator will be managed by OLM.
```

Comment:
```suggestion
2. Create a subscription to the operator. By subscribing to the operator package, the {{prodname}} operator will be managed by OLM.
```

Reply:
Thanks. Done. Btw, applicable to the **Calico openshift** doc too.